spec/system/admin/order_cycles/simple_spec.rb

```diff
@@ -714,7 +714,9 @@
     input = find(".flatpickr-calendar.open .flatpickr-minute")
     input.send_keys datetime.strftime("%M").to_s.strip
     input.send_keys :enter
-    expect(page).to have_content "You have unsaved changes"
+    within "#save-bar" do
+      expect(page).to have_content "You have unsaved changes"
+    end
```

Comment:
The additional `within` can add a bit of execution time, like a sleep, but it probably doesn't do much about the flakiness.

Reply:
Agree. It was not a ground-breaking change, but I thought we could solve it since the issue seems to be about the synchrony between the appearing of the #save-bar and the assertion.

I've tried `wait_until` in https://github.com/openfoodfoundation/openfoodnetwork/pull/9919 . Looks stable locally, but so did the other attempts...